05_vectors_and_result_enum.md

```diff
@@ -9,14 +9,20 @@ fn read_version(transaction_hex: &str) -> u32 {
 ```
 
 There are two different ways we can approach this: 
-1. Read the first 4 pairs of characters and do some base math to determine the u32 integer.
-2. Convert the transaction into a collection bytes first and then just work with the methods available for that type.
+1. read the first 4 pairs of characters and do some base math to determine the u32 integer;
```

Comment:
not familiar with this. Does this need a semicolon at the end?

Reply:
Formally, numbered lists are still single phrases and the colon is not even needed.

The equivalent of 

There are two different ways we can approach this (1) read the first 4 pairs (...), or (2) convert the transaction (...).

Semicolons substitute commas to differentiate those that separate the list elements. It's usual to use them in long items like these even when there are no commas.

See [what APA says](https://apastyle.apa.org/style-grammar-guidelines/lists/lettered) (they are the reference of correct English in the USA for official and academic documents)